packages/lodestar/src/sync/regular.ts

```diff
@@ -32,9 +32,14 @@ export class RegularSync {
   }
 
   public async receiveBlock(block: BeaconBlock): Promise<void> {
-    // TODO: skip block if its a known bad block
-    // skip block if it already exists
     const root = hashTreeRoot(block, this.config.types.BeaconBlock);
+
+    // skip block if its a known bad block
+    if (await this.db.isBadBlockRoot(root)){
+      this.logger.warn("Found bad block, skipping");
```

Comment:
add block root to  log, this is meaningless. Format it to says "received" instead of "found"